core/src/main/java/com/linecorp/armeria/common/CommonPools.java

```diff
@@ -51,8 +51,8 @@
     }
 
     /**
-     * Returns the common blocking task {@link Executor} which is used when
-     * {@link ServerBuilder#blockingTaskExecutor(Executor)} is not specified.
+     * Returns the default common blocking task {@link Executor} which is used for
+     * running a potentially long-running tasks which may block I/O threads.
```

Comment:
nit: used for a potentially long-running? (remove the duplicate running)

Reply:
Fixed in 9576e472ad1ec07311acb045fc2fe29c0a1521e6 (Typo in the commit message though :sob:)